Content.Server.Database/Content.Server.Database.csproj

```diff
@@ -20,5 +20,10 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="6.0.0" />
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="6.0.0-rc.2" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Content.Shared\Content.Shared.csproj" />
+  </ItemGroup>
+
```

Comment:
Yes to pull the LogType enum from Shared since Client, Server and Server.Database use it.
According to PJB doing this has caused some errors in the past but it hasn't caused any for me.